backend/lcfs/web/api/notification/schema.py

```diff
@@ -1,11 +1,39 @@
+from datetime import datetime
 from typing import Any, Dict, List, Optional
 
 from lcfs.db.models.compliance.ComplianceReportStatus import ComplianceReportStatusEnum
 from lcfs.db.models.initiative_agreement.InitiativeAgreementStatus import (
     InitiativeAgreementStatusEnum,
 )
 from lcfs.db.models.transfer.TransferStatus import TransferStatusEnum
-from lcfs.web.api.base import BaseSchema, NotificationTypeEnum
+from lcfs.web.api.base import BaseSchema, NotificationTypeEnum, PaginationResponseSchema
+from pydantic import computed_field, model_validator
+
+
+class OrganizationSchema(BaseSchema):
+    organization_id: int
+    name: str
+
+
+class UserProfileSchema(BaseSchema):
```

Comment:
We should re-use the one from User

Reply:
Creating several schemas that have slightly varying fields with the same names is not maintainable. Lets rename this to NotificationUserSchema or something similar to indicate its purpose.

Reply:
yes, agreed. changed as per suggestion.